packages/bazel-bot/docker-image/Dockerfile

```diff
@@ -14,16 +14,29 @@
 
 # build with:
 # docker build -t gcr.io/repo-automation-bots/bazel-bot .
+
+##########################################################
+# Build the command-line tool that lets us sign JWTs.
+FROM rust:1.49-buster
+RUN git clone https://github.com/mike-engel/jwt-cli.git
```

Comment:
why don't we clone this at a release tag or sha, so that we know the API surface won't change on us.

Reply:
Done.